python/triton/compiler/code_generator.py

```diff
@@ -1,5 +1,6 @@
 import ast
 import inspect
+import numpy as np
```

Comment:
I don't think we want to add a dependency to numpy. Right now triton doesn't depend on numpy (except for tests). It doesn't seem worth adding such a dependency